### DISCUSSION TITLE
how to using on open-tcpnetworkstream? - what's the TransmissionContext wtf?

### DISCUSSION THREAD

**Discussion by NSH531** [PowerShell/PowerShell#18698] [Q&A]
code example:

Open-TCPNetworkStream -TCPClientSocket $TCPClientSocket -ServerIPAddress $ServerIPAddress -TransmissionContext **$TransmissionContext**

**Comment by SeeminglyScience** [comment]
That's not a built-in command (and it gives zero hits on google) so you may want to check where it comes from and ask the author
